Commons/src/main/java/eu/neclab/ngsildbroker/commons/constants/NGSIConstants.java

```diff
@@ -180,4 +180,15 @@
 	public static final String MQTT_VERSION = "mqtt_version";
 	public static final String DEFAULT_DATA_SET_ID = "https://uri.etsi.org/ngsi-ld/default-data-set-id";
 	public final static String QUERY_PARAMETER_DELETE_ALL  = "deleteAll";
+	public static final String NGSI_LD_NOTIFIERINFO = "https://uri.etsi.org/ngsi-ld/default-context/notifierinfo";
+	public static final String NGSI_LD_MQTT_QOS = "https://uri.etsi.org/ngsi-ld/default-context/qos";
+	public static final String NGSI_LD_MQTT_VERSION = "https://uri.etsi.org/ngsi-ld/default-context/version";
+	public static final String DEFAULT_MQTT_QOS = "0";
```

Comment:
[Suggestion] QOS is integer value why we are taking it as string as possible value of QOS is 0,1,2 so better to take it as Integer.